ethcore/account-state/src/substate.rs

```diff
@@ -68,8 +70,9 @@ impl Substate {
 
 #[cfg(test)]
 mod tests {
-	use super::{Substate, Address};
-	use types::log_entry::LogEntry;
+	use ethereum_types::Address;
+	use common_types::log_entry::LogEntry;
```

Comment:
Was the idea to sort them? Seems sort before rename didn't work that good.

Reply:
I think it's my IDE that first groups imports in std, external crates, local crates and then sorts each group, but yeah looks odd.